pkgs/development/libraries/cxxopts/default.nix

```diff
@@ -17,11 +17,14 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  # Conflict on case-insensitive filesystems.
+  dontUseCmakeBuildDir = true;
```

Comment:
Why not provide an alternative `cmakeDir`?

Reply:
This just seemed more concise and doesn't fail. I think an alternate dir would look like:

```nix
{
  dontUseCmakeBuildDir = true;
  preConfigure = ''
    mkdir _build
    cd _build
    cmakeDir=..
  '';
}
```

> Maybe you can ask upstream to make a release with those fixes?

I'm not familiar with these packages to know what that would do with ydotool.

> Anyway, I think we should rather mark it as broken.

Good point, I made a change for this.

Reply:
> I think an alternate dir would look like

Never mind, yours looks definitely cleaner.



> I'm not familiar with these packages to know what that would do with ydotool.

Let's hope @WilliButz can give some advice.

Reply:
```
{
  dontUseCmakeBuildDir = true;
  preConfigure = ''
    mkdir _build
    cd _build
    cmakeDir=..
  '';
}
```
That's essentially what the cmake hook does anyway. I think `dontUseCmakeBuildDir = true;` is fine. Shouldn't affect installation.... dependning on upstream's logic.